recipe/meta.yaml

```diff
@@ -91,6 +92,8 @@ outputs:
 #  it has been decided to skip the builds for all of Lumen's additional AI outputs.
 
   - name: {{ name|lower }}-ai
+    build:
+      skip: true  # [py<310]
     requirements:
       host:
         - python
```

Comment:
Missing dependency semchunk https://github.com/holoviz/lumen/blob/v0.10.1/pyproject.toml#L53

Missing panel-graphic-walker[kernel] dependencies gw-dsl-parser and pygwalker https://github.com/panel-extensions/panel-graphic-walker/blob/v0.6.4/pyproject.toml#L85-L86

Reply:
Where should the panel-graphic-walker[kernel] dependencies be placed?

Reply:
Next to `panel-graphic-walker`. Conda does not understand [kernel], so we need to expand it ourselves.